File: java/program/Perusahaan.java
public class Perusahaan extends InformasiKontak {
    // Class ini menurunkan InformasiKontak, jadi otomatis memiliki atribut dari parent class.
    // Lalu saya tambahin atribut khusus buat perusahaan.
    private String nama;
    private int tahunBerdiri;
    private String bidangUsaha;

    /* Ini constructor. Jadi pas kita bikin objek Perusahaan, kita bisa langsung isi data kontak (lewat super)
    dan juga data khusus perusahaan. */
    public Perusahaan(String noHp, String email, String alamat,
                      String nama, int tahunBerdiri, String bidangUsaha) {
        super(noHp, email, alamat); // manggil constructor dari parent (InformasiKontak)
        this.nama = nama;
        this.tahunBerdiri = tahunBerdiri;
        this.bidangUsaha = bidangUsaha;
    }

    // Getter dan setter di bawah ini fungsinya buat ambil atau ubah data.
    public String getNama() {
        return nama;
    }

    public void setNama(String nama) {
        this.nama = nama;
    }

    public int getTahunBerdiri() {
        return tahunBerdiri;
    }

    public void setTahunBerdiri(int tahunBerdiri) {
        this.tahunBerdiri = tahunBerdiri;
    }
    
    public String getBidangUsaha() {
        return bidangUsaha;
    }

    public void setBidangUsaha(String bidangUsaha) {
        this.bidangUsaha = bidangUsaha;
    }
}

File: python/program/Perusahaan.py
from InformasiKontak import InformasiKontak

# subclass Perusahaan
class Perusahaan(InformasiKontak):

    def __init__(self, noHp: str, email: str, alamat: str,
                 nama: str, tahunBerdiri: int, bidangUsaha: str):
        # panggil constructor parent
        super().__init__(noHp, email, alamat)
        # atribut khusus Perusahaan
        self._nama = str(nama)
        self._tahunBerdiri = int(tahunBerdiri)
        self._bidangUsaha = str(bidangUsaha)

    # Getter & Setter Nama
    def getNama(self) -> str:
        return self._nama
    
    def setNama(self, nama: str) -> None:
        self._nama = str(nama)

    # Getter & Setter Tahun Berdiri
    def getTahunBerdiri(self) -> int:
        return self._tahunBerdiri
    
    def setTahunBerdiri(self, tahunBerdiri: int) -> None:
        self._tahunBerdiri = int(tahunBerdiri)

    # Getter & Setter Bidang Usaha
    def getBidangUsaha(self) -> str:
        return self._bidangUsaha
    
    def setBidangUsaha(self, bidangUsaha: str) -> None:
        self._bidangUsaha = str(bidangUsaha)
